soc/x86/lakemont/linker.ld

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) 2011-2014, Wind River Systems, Inc.
+ *
```

Comment:
missing Copyright (c) 2021 Intel Corporation

Reply:
This is a straight copy from `soc/x86/atom/linker.ld` without any modifications, hence no new copyright line.

Reply:
Oh ok, thank you!